pwn/toplevel.py

```diff
@@ -89,3 +89,13 @@
 
 # Equivalence with the default behavior of "from import *"
 # __all__ = [x for x in tuple(globals()) if not x.startswith('_')]
+
+
+def condition(data):
+    if b"e" in data:
+        return True
+    else:
+        return False
+
+def condition_alway_false(data):
```

Comment:
```suggestion
def condition_always_false(data):
```
typo

Why is this even necessary?

Reply:
This was for unit tests but this sould actually move to the tests directly instead of keep these to this file. I will remove them.